src/protagonist/API/Features/NamedQueries/Requests/GetAllNamedQueries.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using API.Infrastructure.Requests;
+using DLCS.Model.Assets.NamedQueries;
+using DLCS.Repository;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.Features.NamedQueries.Requests;
+
+public class GetAllNamedQueries : IRequest<FetchEntityResult<IReadOnlyCollection<NamedQuery>>>
+{
+    public int CustomerId { get; }
+    
+    public GetAllNamedQueries(int id)
+    {
+        CustomerId = id;
+    }
+}
+
+public class GetAllNamedQueriesHandler : IRequestHandler<GetAllNamedQueries, FetchEntityResult<IReadOnlyCollection<NamedQuery>>>
+{
+    private readonly DlcsContext dbContext;
+    
+    public GetAllNamedQueriesHandler(DlcsContext dbContext)
+    {
+        this.dbContext = dbContext;
+    }
+    
+    public async Task<FetchEntityResult<IReadOnlyCollection<NamedQuery>>> Handle(
+        GetAllNamedQueries request, 
+        CancellationToken cancellationToken)
+    {
+        var namedQueries = await dbContext.NamedQueries
+            .AsNoTracking()
+            .Where(nq => nq.Customer == request.CustomerId || nq.Global == true)
```

Comment:
`nq.Global == true` is redundant as `Global` is a boolean, when the prop is `bool` rather than `bool?` it's idiomatic to omit `== true`.

All you need is `.Where(nq => nq.Customer == request.CustomerId || nq.Global)` (same with `GetNamedQuery`

Reply:
Changed in 0549da0d4624b13085c61d71077f370cf6734abc 👍